### ISSUE TITLE
Committee meeting notes Dec 19

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/london-membership#77]
Present: @NataliaLKB @claireinez and me

Action points:
- Course Facilitator checklist: code of conduct and complaints form
- Post our suggested changes to membership criteria and park a PR ready for the 2018 AGM

I think that was it.

**Comment by NataliaLKB** [comment]
Yes that sounds right to me! Are you doing both actions @sofer ?
